bin/generate-configs.js

```diff
@@ -38,6 +40,7 @@ async function main() {
       hpMasternodesAmdCount,
       hpMasternodesArmCount,
       mixerCount,
+      loadTestCount,
```

Comment:
Generate config doesn't set metrics (let's set always 1 into config) and load test nodes count

Reply:
Fixed, I've removed them from /bin/generate because it's getting too complicated. If someone wants to change they can change the tfvars